gnd/src/main/java/com/google/android/gnd/ui/common/BottomSheetBehavior.java

```diff
@@ -410,7 +410,7 @@ public void onNestedPreScroll(
       return;
     }
     View scrollingChild = nestedScrollingChildRef.get();
-    if (target.equals(scrollingChild)) {
+    if (!target.equals(scrollingChild)) {
```

Comment:
Is `equals()` the same as `==` for views? We really wanted to ensure it was the same instance. Is this correct?

Reply:
Yes, it has the same effect.

Reply:
Sgtm!